wolfcrypt/src/wc_port.c

```diff
@@ -1468,6 +1556,37 @@ int wolfSSL_CryptHwMutexUnLock(void) {
         return BAD_MUTEX_E;
     }
 
+#elif defined(WOLFSSL_ZEPHYR)
+
+    int wc_InitMutex(wolfSSL_Mutex* m)
+    {
+        k_mutex_init(m);
+
+        return 0;
+    }
+
+    int wc_FreeMutex(wolfSSL_Mutex* m)
+    {
+        return 0;
```

Comment:
No resource free needed for the mutex?

Reply:
No free API for K_mutex - no need to free.